src/dualscreen.tsx

```diff
@@ -334,17 +338,20 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
         get: () => (
           <ResponsiveManager onDefaultSize={this._setMode}>
             <PipMinimized
-              show={() => this._switchToPIP()}
+              show={(byKeyboard: boolean) => this._switchToPIP(Animations.None, getValueOrUndefined(byKeyboard, Labels.Hide))}
               player={this._getSecondaryPlayer()}
-              onInverse={() => this._switchToSingleMediaInverse(Animations.Fade)}
+              onInverse={(byKeyboard: boolean) =>
+                this._switchToSingleMediaInverse(Animations.Fade, getValueOrUndefined(byKeyboard, Labels.SwitchScreen))
+              }
+              focusOnMount={focusOnMount}
             />
           </ResponsiveManager>
         )
       })
     );
   };
 
-  private _switchToSingleMediaInverse = (parentAnimation: Animations = Animations.None) => {
+  private _switchToSingleMediaInverse = (parentAnimation: Animations = Animations.None, focusOnMount?: Labels) => {
```

Comment:
same here. How did it pass TS? should be boolean

Reply:
it's not mistake.
https://github.com/kaltura/playkit-js-dual-screen/pull/49/files#diff-03f4f6934e0494b7371fce31f7a2bbf3cd046c3e69a761f3885537dd5abbc831R60
https://github.com/kaltura/playkit-js-dual-screen/pull/49/files#diff-03f4f6934e0494b7371fce31f7a2bbf3cd046c3e69a761f3885537dd5abbc831R73
etc.
Only SbS has `boolean` because only 1 button might be focused (left expand button)
for PiP, PiPinverse we have 3 buttons, for SingleMedia, SingleMediaInvers  2 buttons that might be selected